osi_sensorview.proto

```diff
@@ -148,6 +148,10 @@ message SensorView
     // Ultrasonic-specific SensorView(s).
     //
     repeated UltrasonicSensorView ultrasonic_sensor_view = 1004;
+    
+    // Raytracer-specific View(s).
+    //
+    repeated RaytracerView raytarcer_view = 1005;
```

Comment:
As more as I think about it, I even assume `repeated RayTracerView ray_tracer_view` would be correct.

And if this is correct, this would mean to write `ray tracing` and `ray tracer` separated all over the file, not?

Reply:
Absolutely right. Changes done in osi_sensorviewconfiguration and osi_sensorview